repositories.yaml

```diff
@@ -159,21 +159,37 @@ libraries:
         branch: main
         dependency_name: nimbus
 
-# See https://mozilla.github.io/probe-scraper/#tag/application
-applications:
-  - app_name: firefox_desktop
-    canonical_app_name: Firefox for Desktop
-    app_description: The desktop version of Firefox
-    url: https://github.com/mozilla/gecko-dev
+  - library_name: gecko
+    description: The browser engine developed by Mozilla
```

Comment:
Can we mention something here about how this compares with the geckoview library? Ideally in the description for both.

Reply:
There's a geckoview library? Oh, you mean `engine-gecko`.

That's a tricky one to put into words. `engine-gecko` is currently being used to show which applications use [GeckoView Streaming Telemetry (EXTRACT)](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/internals/geckoview-streaming.html) (which is just Fenix). In the not-too-distant future we hope to migrate everyone one of those to be collected in Gecko and reported in all things that use both Glean and Gecko (removing the need for `engine-gecko` to exist in `repositories.yaml` entirely). In fact, Jan-Erik's already started that process.

So, in terms of `repositories.yaml`, `gecko`'s relationship with `engine-gecko` is one of temporary fraternity. They don't really have anything to do with each other, and we can hopefully remove `engine-gecko` from the file within a month.

This leaves me uncertain about how to write these. Do you have any ideas @wlach ?

Reply:
I see. If `engine-gecko` is going away soon, maybe we can just leave things as-is.